apps/gallery/js/ImageEditor.js

```diff
@@ -298,10 +300,16 @@ function editPhoto(n) {
     var imagesize = metadata.width * metadata.height;
     var maxsize = CONFIG_MAX_EDIT_PIXEL_SIZE || CONFIG_MAX_IMAGE_PIXEL_SIZE;
 
+    LazyLoader.load(['shared/js/media/jpeg-exif.js'], () => {
```

Comment:
Also, maybe you should defer this code until after you've run the code below that might resize the image. Otherwise you might have cropResizeRotate() and this code both parsing exif data at the same time, which would use more memory than desireable.